python/cudf/cudf/core/dataframe.py

```diff
@@ -3275,7 +3195,8 @@ def _create_output_frame(data, percentiles=None):
                 {
                     col: data[col].describe(percentiles=percentiles)
                     for col in data.columns
-                }
+                },
+                index=Series([]).describe(percentiles=percentiles).index,
```

Comment:
use empty column constructor instead of empty list

Reply:
Done